src/coreclr/jit/gentree.h

```diff
@@ -5323,6 +5323,34 @@ struct GenTreeJitIntrinsic : public GenTreeMultiOp
         return (CorInfoType)gtSimdBaseJitType;
     }
 
+    CorInfoType GetNormalizedSimdBaseJitType() const
+    {
+        CorInfoType simdBaseJitType = GetSimdBaseJitType();
+        switch (simdBaseJitType)
+        {
+            case CORINFO_TYPE_NATIVEINT:
+            {
+#if TARGET_64BIT
```

Comment:
Nit: We generally use `#ifdef` for these defines.

```suggestion
#ifdef TARGET_64BIT
```

Reply:
(More of these below that I won't comment on)

Reply:
Sure, makes sense.